playwright-helpers/config.ts

```diff
@@ -0,0 +1,11 @@
+export const config = {
+  appUrl: 'http://localhost:9001/app/',
+  accountLoginOrigin: 'https://www.ovh.com',
+  ovhLogout: 'https://www.ovh.com/auth/?action=disconnect',
+  ovhLoginOrigin: 'https://login.corp.ovh.com',
+  accountLogin: 'accountLogin',
+  email: 'your-ovh-mail@corp.ovh.com',
+  password: 'your-password',
```

Comment:
```suggestion
  password: '',
```

Maybe try an empty string to avoid the sonar error ?